components/infieldbutton/metadata/infieldbutton.yml

```diff
@@ -22,7 +26,7 @@ examples:
           <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">S</h4>
           <button class="spectrum-InfieldButton spectrum-InfieldButton--sizeS spectrum-InfieldButton--right" aria-haspopup="listbox">
             <div class="spectrum-InfieldButton-fill">
-              <svg xmlns="http://www.w3.org/2000/svg" class="spectrum-Icon spectrum-InfieldButton-icon" focusable="false" aria-hidden="true" aria-label="Add">
+              <svg xmlns="http://www.w3.org/2000/svg" class="spectrum-Icon spectrum-Icon--sizeS spectrum-InfieldButton-icon" focusable="false" aria-hidden="true" aria-label="Add">
```

Comment:
noticing this will result in an empty button. do we want to do something similar as we did with [ActionButton](https://github.com/adobe/spectrum-css/pull/1815)?

Reply:
Oh good call out. I'll add some aria attributes here